### DISCUSSION TITLE
Pathway I created was removed

### DISCUSSION THREAD

**Discussion by dr-oxylipin** [wikipathways/wikipathways-help#133] [Q&A] [answered]
A pathway I created (WP5141) was removed, and I am not sure why. I suspect it was due to some nodes not having annotations available. There are no similar pathways on WP, and I need to upload the pathway to Cytoscape. WP5141 is available on Pathvisio, but not WP nor Cytoscape. I also have the original GPML file, so I could create a new pathway at the sake of redundancy. Any guidance on how to get it back would be appreciated.

**Comment by khanspers** [comment, marked as answer]
Hi @dr-oxylipin, you're probably right abut the pathway being removed because of missing data node annotations. If it's possible to add annotations that would be ideal. For now, your pathway is still available on the classic WikiPathways site: https://classic.wikipathways.org/index.php/Pathway:WP5141

There is a workaround for importing local gpml files into Cytoscape. If you first open _any_ pathway using the WikiPathways app, you can then open a local pathway via "File > Import > Network from File".
